client/dom/select2Terms.js

```diff
@@ -1,7 +1,7 @@
 import { showTermsTree } from '../mass/groups'
 import { Menu } from '#dom/menu'
 
-export function addDynamicScatterForm(tip, app) {
```

Comment:
please add a comment this is an "intermediate" interface that is used by multiple chart buttons and will route to different apps

Reply:
Done